packages/core/src/tests/utils/test/page.ts

```diff
@@ -130,8 +130,8 @@ export const regressionTest = testBase.extend<{
     page = await extendPageFixture(page, testInfo);
 
     await page.route('*/**/svg/*.svg', async (route, request) => {
-      if (!process.env.CI) {
-        const [__, svg] = request.url().split('/svg/');
+      if (process.env.CI !== 'true') {
```

Comment:
This check here is not correct. The variable CI is no necessary "true" could also be "1"
`CI=1 pnpm test`
as it was before was totally correct.